Sources/Graphaello/Extensions/ExprSyntax+erasure.swift

```diff
@@ -4,8 +4,8 @@ import SwiftSyntax
 
 extension ExprSyntax {
 
-    func withoutErasure() -> ExprSyntaxProtocol {
-        return asProtocol(ExprSyntaxProtocol.self)
+    func switchOver() -> SyntaxEnum {
```

Comment:
Some background on why `as(SyntaxEnum.self)` is only defined on `Syntax` and not on `ExprSyntax`: The `SyntaxEnum` contains every syntax type, while `ExprSyntax` already has the set restricted, thus we decided that it was better to make the "upcast" explicit by requiring to write `as(Syntax.self).as(SyntaxEnum.self)`.

Reply:
Very good point